### PR TITLE
Dependabot auto updates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  # Enable version updates for cargo
+  - package-ecosystem: "cargo"
+    # Look for 'Cargo' files in the `root` directory
+    directory: "/"
+    # Check for updates daily
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "willemml"
+      - "MixedMatched"
```
Comment (willemml) [inline .github/dependabot.yml]: MixedMatched is not a codeowner for this repo yet.

**Review by willemml — CHANGES_REQUESTED** [review]
I am hesitant to add auto updates (even though they will require PR review) because we don't have unit testing (or any testing for that matter) yet.

**Comment by ghost** [comment]
@willemml We may want to see if we can use RedGl0w/numworks-qemu to create a test workflow. See issue #nw-rs/rustworks/issues/16

**Comment by RedGl0w** [comment]
It still needs lot of improvement, mainly on the external flash and the stm32f730

**Comment by ghost** [comment]
> It still needs lot of improvement, mainly on the external flash and the stm32f730

But, does it boot n0110?

**Comment by RedGl0w** [comment]
Nope, n0110 isn't a thing in the emulator yet, because there isn't yet any flash emulation. However, most of the other thing should already be ready (the screen driver emulates pushing / pulling (thanks to boricj), there's emulation for crc, rng,...)
And so it shouldn't be that hard to enable n0110 emulation

**Comment by willemml** [comment]
This has been enabled at organization level (minus automatic PRs, which may be done later if/when the project stabilizes and hits release status with tests).

**Pull request closed without merging**